src/EA.Weee.RequestHandlers.Tests.DataAccess/Search/FetchOrganisationSearchResultsForCache/FetchOrganisationSearchResultsForCacheDataAccessTests.cs

```diff
@@ -275,42 +278,68 @@ public async Task FetchCompleteOrganisations_WithSeveralResults_ReturnsResultsOr
             }
         }
 
-        /// <summary>
-        /// Ensures that search results representing organisations which are partnerships will use
-        /// the 'TradingName' column from the database as the organisation name.
-        /// </summary>
-        /// <returns></returns>
         [Fact]
-        public async Task FetchCompleteOrganisations_OrganisationHasNoScheme_OrganisationReturned()
+        public async Task FetchCompleteOrganisations_OrganisationHasNoScheme_OrganisationNotReturned()
```

Comment:
This looks like it tests where the Organisation has no related entities, I think :) If so can we rename otherwise all looks good

Reply:
Good point, renamed